### PR TITLE
code cleanup and formatting corrections in iscsi package.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

> /kind cleanup

-->
```release-note
NONE
```

### CODE REVIEW

File: pkg/iscsi/driver.go [hunk 1]
```diff
@@ -54,7 +50,7 @@ func NewDriver(nodeID, endpoint string) *driver {
 		endpoint: endpoint,
 	}
 
-	if err := os.MkdirAll(fmt.Sprintf("/var/run/%s", driverName), 0755); err != nil {
+	if err := os.MkdirAll(fmt.Sprintf("/var/run/%s", driverName), 0o755); err != nil {
```
Comment (andyzhangx) [inline pkg/iscsi/driver.go:53]: `0o755` works? people usually use `0755` in golang code

Comment (humblec) [reply pkg/iscsi/driver.go:53]: yeah, it works and we do use that in our CSI driver.

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**